_posts/en/2019-10-15-case-continuous-improvement-deeper-dive.md

```diff
@@ -19,11 +19,11 @@ Luckily with desmos the heavy lifting is done for us.
 If you haven't already done so, click [here](https://www.desmos.com/calculator/ouestfova3) for an example.
 You should see something as follows
 
-![Area between curves]({{ site.baseurl }}/assets/images/areaundercurve.PNG)
+![The image shows the correlation between how much time a person can loose when choosing to invest in continuous learning. The first line represents 20% investment of time for a 10% trade off. The second line represents 30% investment of time for a 20% trade off.]({{ site.baseurl }}/assets/images/areaundercurve.PNG)
 
```

Comment:
I would describe this image a bit differently.....e.g. this image is a 2D line chart with an x axis going from 0 to 8 and a y axis going from 0 to 8. The there are two lines on the chart; a red line and a blue line. Both start at point 0,0. The blue line is straight and ends at point x=8, y=8. The red line is curved and ends at x=~7.75, y=8. The two lines intersect at point 6.192. The area under the curve (between points 0,0 and the intersection at point 6.192) is coloured in green.

Reply:
Or something like that. :-)